src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -195,6 +195,12 @@ public function getEnv($prefix, $name, \Closure $getEnv)
             return str_getcsv($env);
         }
 
+        if ('nullable' === $prefix) {
+            $nullables = array('', 'null', 'NULL', 'Null');
```

Comment:
IMHO only `""` is a nullable value

Reply:
In some cases having an explicit "null" string may help:

- Some cloud providers might refuse an empty string when configuring env vars
- Some developers may want to explicitely set a variable to null to avoid the "damn, did I forget to fill that one?" ambiguity

Anyway we assume the developer using the `nullable:` processor is aware of the possible nullable values (this will be stated in the docs) and thus can't be mistaken.

Reply:
but it disables using `null` as possible expected value :confused:

Reply:
What do you mean?

Reply:
Alright, I get your point. But given [what I exposed earlier](https://github.com/symfony/symfony/pull/29767#discussion_r245280034), I don't know what to choose.  

Poke @nicolas-grekas (you're authored on that class) @dunglas (you worked on this too), any opinion on this?

Reply:
Why? If you don’t use the nullable processor explicitly, nothing changes and you can use empty string

Reply:
I also agree with @ro0NL

Reply:
Alright, let me do the necessary changes.